tests/components/zha/zha_devices_list.py

```diff
@@ -3013,11 +3013,6 @@
                 DEV_SIG_ENT_MAP_CLASS: "Illuminance",
                 DEV_SIG_ENT_MAP_ID: "sensor.lumi_lumi_sensor_motion_aq2_illuminance",
             },
-            ("sensor", "00:11:22:33:44:55:66:77-1-1"): {
```

Comment:
Why did Ruff remove this entry? Unit tests seem to pass without it.

Reply:
It is automatically removed by Ruff, as it is a duplicate:

![CleanShot 2023-07-23 at 21 52 58](https://github.com/home-assistant/core/assets/195327/18a317b9-2339-4b7c-aef6-c78280ebe950)